src/components/DataManagerLogDetails.vue

```diff
@@ -288,9 +286,10 @@ section {
 .header {
   display: grid;
   grid-template-columns: repeat(2, 1fr);
+  grid-gap: var(--spacer-sm);
 }
 
-.config {
+.config-details {
   align-self: end;
```

Comment:
I don't think this is needed. The content should instead expand to fill the remaining space. both grid columns should take equal space. The Width of the grid should be restricted to control the width of its children.

Reply:
The align-self property is aligning the grid column to the end vertically, not horizontally, as the layout suggests. I used this property to position the configuration name section at the bottom vertically. After removing it, this is how it looks:

![Screenshot from 2024-10-18 11-06-48](https://github.com/user-attachments/assets/4f0c5037-9c8f-497c-9836-399ae6de8fe5)

Reply:
makes sense, I mixed up justify and align. Thanks for clarifying. It was correct the way you had it